statsmodels/stats/_lilliefors.py

```diff
@@ -216,21 +308,37 @@ def kstest_normal(x, pvalmethod='approx'):
     '''
 
     x = np.asarray(x)
-    z = (x-x.mean())/x.std(ddof=1)
-    nobs = len(z)
+    nobs = len(x)
+
+    if dist == 'norm':
+        z = (x-x.mean())/x.std(ddof=1)
+        test_d = stats.norm.cdf
+        lilliefors_table = get_lilliefors_table(dist='norm')
+    elif dist == 'exp':
+        z = x
+        ex = stats.expon(scale=1./x.mean())
```

Comment:
there is a difference in the scaled and unscaled "space", however these two should be equal
stats.expon(scale=1/(1./X_scaled.mean())).cdf(x) ==  stats.expon().cdf(x/X_scaled.mean()) 

the latter is 
stats.expon().cdf(z)   for z = x/X_scaled.mean() 
we I think we use